packages/core/src/toolkit-provider/ToolkitProvider.tsx

```diff
@@ -90,6 +89,7 @@ interface ToolkitProviderThatAppliesClassesToChild {
 }
 
 type ThemeNameType = string | Array<string>;
+
 interface ToolkitProviderThatInjectsThemeElement {
   children: ReactNode;
```

Comment:
Not a big deal but maybe children should be optional

Reply:
Potentially but I think this was just to match the existing behaviour. I would expect the ThemeProvider to always have children